### ISSUE TITLE
plotmafSummary error (  j out of bounds )

### ISSUE THREAD

**Issue by hmkim** [PoisonAlien/maftools#9]
Hi, All

When I try to use the maftools with converted maf format from mutect vcf (vcf2maf), I got the error i n below.

```
laml2 = read.maf(maf = '~/Downloads/TN1508R0520-1_TN1508R0519-1.maf', removeSilent = T, useAll = T)
```

[TN1508R0520-1_TN1508R0519-1.maf.zip](https://github.com/PoisonAlien/maftools/files/430580/TN1508R0520-1_TN1508R0519-1.maf.zip)

```
plotmafSummary(maf = laml2,  addStat = 'median', dashboard = TRUE)
Error in `[.data.table`(vt.plot.dat, , c(2:4), with = FALSE) : 
  j out of bounds
```

```
> laml2
An object of class  MAF 
                       ID summary Mean Median
1:             NCBI_Build  GRCh37   NA     NA
2:                 Center       .   NA     NA
3:                Samples       1   NA     NA
4:                 nGenes     493   NA     NA
5:      Missense_Mutation     507  507    507
6:      Nonsense_Mutation      22   22     22
7:            Splice_Site       1    1      1
8: Translation_Start_Site       2    2      2
9:                  total     532  532    532
```

Can you check it please?

**Comment by PoisonAlien** [comment]
Hi,
Thanks for reporting. This is a bug in 0.99.34. 

I have fixed it in 0.99.40. Can you install recent version from github (should be up on bioconductor soon) and try it ? 

But please remember that your data has only single sample, and you cant do much of analysis or plotting with single sample. I would suggest you to put all samples into a single MAF and process it.

**Comment by hmkim** [comment]
@PoisonAlien Thanks for youre reply. I'll try to run the analysis with recent version.

and I just testing with single sample data. When it finished, I'll play with multi sample data.

**Comment by hmkim** [comment]
This fixed it in 0.99.40.
